SteamKit2/SteamKit2/Steam/Handlers/SteamAuthTicket/Callbacks.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using SteamKit2.Internal;
+
+namespace SteamKit2
+{
+    public sealed partial class SteamAuthTicket
+    {
+        /// <summary>
+        /// This callback is fired when Steam accepts our auth ticket as valid.
+        /// </summary>
+        internal sealed class TicketAcceptedCallback : CallbackMsg
+        {
+            /// <summary>
+            /// <see cref="List{T}"/> of AppIDs of the games that have generated tickets.
+            /// </summary>
+            public List<uint> AppIDs { get; private set; }
+            /// <summary>
+            /// <see cref="List{T}"/> of CRC32 hashes of activated tickets.
+            /// </summary>
+            public List<uint> ActiveTicketsCRC { get; private set; }
+            /// <summary>
+            /// Number of message in sequence.
+            /// </summary>
+            public uint MessageSequence { get; private set; }
+
+            internal TicketAcceptedCallback( JobID jobId, CMsgClientAuthListAck body )
+            {
+                JobID = jobId;
+                AppIDs = body.app_ids;
+                ActiveTicketsCRC = body.ticket_crc;
+                MessageSequence = body.message_sequence;
+            }
+        }
+
+        /// <summary>
+        /// This callback is fired when generated ticket was successfully used to authenticate user.
+        /// </summary>
+        internal sealed class TicketAuthCompleteCallback : CallbackMsg
```

Comment:
Make these callbacks public, as youre using `Client.Post`, otherwise consumers won't be able to use them.